woocommerce/woocommerce-admin
# Homepage: Add stats overview selection

Partially solves https://github.com/woocommerce/woocommerce-admin/issues/4084

This PR starts the Stats Overview component by adding a card and the stat selection logic. More logic, CSS and content will be added in a follow up. Jetpack stuff is omitted for now.

![Screen Shot 2020-05-01 at 2 58 47 PM](https://user-images.githubusercontent.com/1922453/80778506-bda45f00-8bbc-11ea-95c0-27a88f05caf6.png)

### Detailed test instructions:

1. Go to the Homepage.
2. Note the defaults and ones selected already match what is described in https://github.com/woocommerce/woocommerce-admin/issues/4084
2. Toggle a stat (note a very slow refresh, more on that below).
3. Refresh the page and verify your selection persists.

### Note on super slow update of toggle

Something is going on with wc-api and Fresh Data that makes this super slow to update. The Dashboard gets around this by keeping a duplicate copy of user preference data in state and updating that first. I think this is a mistake, but can be added later if need be. The better solution would be to refactor `wc-api/user` to a wp.data dataStore. I'd like to follow up on that.
